providers/gitlab.go

```diff
@@ -32,7 +35,7 @@ func NewGitLabProvider(p *ProviderData) *GitLabProvider {
 	p.ProviderName = "GitLab"
 
 	if p.Scope == "" {
-		p.Scope = "openid email"
+		p.Scope = "openid email read_api"
```

Comment:
What's the best approach for this ? I can't setup a condition inside New, because there is no info about projects at this time in the execution. In the github provider, there is method to set group https://github.com/oauth2-proxy/oauth2-proxy/blob/master/providers/github.go#L95, I can duplicate this behavior in the gitlab provider and ensure in this method that "read_api" will be added to scope only once. Thanks for reply.